### DISCUSSION TITLE
Going into tags, editing a transaction - doesn't go back to tags

### DISCUSSION THREAD

**Discussion by bartekd123** [firefly-iii/firefly-iii#5194] [Firefly III]
I noticed in 5.6.2, when I am in tags, and I edit a transaction, and then hit save, it brings me to the dashboard now. It used to bring me back to the tags.

Is this change on purpose, or it is just something on my system?

**Comment by JC5** [comment]
It happens to me as well, not sure why. I'll check it out :+1:

**Reply by bartekd123** [reply]
Thanks James!

**Reply by bartekd123** [reply]
Hey James, did you figure this one out? Anything i can do to help, like create a bug instead of having it in the discussion. Its not a huge deal, but I import all my transactions, then go through the tagged ones, and manually change a few things. Its an extra couple clicks when going through each of my transactions

**Reply by JC5** [reply]
It's on the list. I wasn't able to replicate it at some moment earlier but I am checking it out.

**Reply by JC5** [reply]
This is still an issue. I don't know the exact cause yet. Since I'm working on a new layout, I'm afraid that for the time being, you (and me) will have to live with it. Thanks for your patience in this.
